docs/workflow_tasks/queue_granules.md

```diff
@@ -0,0 +1,94 @@
+---
+id: queue_granules
+title: Queue Granules
+hide_title: false
+---
+
+This task utilizes the Cumulus Message Adapter to interpret and construct incoming and outgoing messages.
+
+Links to the npm package, task input, output and configuration schema definitions, and more can be found on the auto-generated [Cumulus Tasks](../tasks) page.
+
+## Summary
+
+The purpose of this task is to schedule ingest of granules that were discovered on a remote host, whether via the [DiscoverGranules](./discover_granules) task or the [ParsePDR](./parse_pdr) task.
+
+The task utilizes a defined [collection](../data-cookbooks/setup#collections) in concert with a defined [provider](../data-cookbooks/setup#providers), either on each granule, or passed in via config to queue up ingest executions for each granule, or for batches of granules.
+
+The constructed granules object is defined by the collection passed in the configuration, and has impacts to other provided core [Cumulus Tasks](../tasks).
+
+Users of this task in a workflow are encouraged to carefully consider their configuration in context of downstream tasks and workflows.
+
+## Task Inputs
+
+Each of the following sections are a high-level discussion of the intent of the various input/output/config values.
+
+For the most recent config.json schema, please see the [Cumulus Tasks page](../tasks) entry for the schema.
+
+### Input
+
+This task expects an incoming input that contains granules and information about them and their files. For the specifics, see the [Cumulus Tasks page](../tasks) entry for the schema.
+
+This input is most commonly the output from a preceding [DiscoverGranules](./discover_granules) or [ParsePDR](./parse_pdr) task.
+
+### Cumulus Configuration
+
+This task does expect values to be set in the `task_config` CMA parameters for the workflows.  A schema exists that defines the requirements for the task.
+
+For the most recent config.json schema, please see the [Cumulus Tasks page](../tasks) entry for the schema.
+
+Below are expanded descriptions of selected config keys:
+
+#### Provider
+
+A Cumulus [provider](https://github.com/nasa/cumulus/blob/master/packages/api/models/schemas.js) object.  Used to define connection information for a location to scan for granule discovery.
```

Comment:
Do we want to point at the Dynamo schema? I guess it's still valid as of `master`

Reply:
For now I think it is the best option. We can put a comment on the doc ticket to note this sort of thing, or file a new ticket.